openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1326,7 +1326,11 @@ To sender-constrain Access Tokens, see the recommendations in Section 4.10.1 in
 
 # Privacy Considerations
 
-TBD
+## Data Minimization in the Issued Credential
+
+It is on purpose that the Credential Offer does not contain `credentialSubject` nor `claims` properties (other Credential Format Profiles may define other names for this property), while Authorization Details and Credential Request do. This is because these properties are meant to be used by the Wallet to specify which claims it is requesting to be issued out of all the claims the Credential Issuer is capable of issuing for this particular credential. Credential Offer is a mere "invitation" from the Credential Issuer to the Wallet to start the issuance flow.
```

Comment:
following other/previous chat on this, I really like the snake case in all the oauth2/openid specs, then `credentialSubject` should became `credential_subject`

A study demonstrates that snake case is more readable: https://www.cs.kent.edu/~jmaletic/papers/ICPC2010-CamelCaseUnderScoreClouds.pdf

we already have snake case in the oauth2/jwt ecosystem and I really would like to preserve it :-)

Reply:
`credentialSubject` is what is used in VCDM, it is `claims` for non-VCDM credential formats  such as[sd-jwt vc](https://github.com/vcstuff/oid4vc-haip-sd-jwt-vc/pull/56/files) and mdocs.
from a processing perspective, I think it is easier if wallet/issuer can just take the credentialSubject object in VCI and map it to the VCDM compliant payload.

if you disagree, please open a separate issue on this, it is out of scope for this PR.